pkg/models/setting.go

```diff
@@ -1,15 +1,24 @@
-package client
+package models
 
 import (
 	"encoding/json"
 	"fmt"
+
 	"github.com/grafana/grafana-aws-sdk/pkg/awsds"
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 )
 
+const EDGE_REGION string = "Edge"
```

Comment:
Nit: I saw some place is using `models.EDGE_REGION`, should we keep consistent with this.